geometry/geometry-engine-simplify/src/main/java/com/esri/samples/geometry_engine_simplify/GeometryEngineSimplifySample.java

```diff
@@ -119,7 +124,8 @@ public void start(Stage stage) {
       // add buttons to the control panel
       controlsVBox.getChildren().addAll(simplifyButton, resetButton);
 
-      map = new ArcGISMap(Basemap.createLightGrayCanvas());
+      // create a map with a basemap style
+      map = new ArcGISMap(BasemapStyle.ARCGIS_LIGHT_GRAY);
 
       // enable geometry operations when ArcGISMap is done loading
```

Comment:
A suggestion for plain english
```suggestion
      // enable geometry operations when the map is done loading
```